tests/test_stamps.py

```diff
@@ -46,15 +49,32 @@ def make_stamps(n_stamps=3):
         stampVarArray += 1000.
     ras = np.random.rand(n_stamps)*360.
     decs = np.random.rand(n_stamps)*180 - 90
+    archive_elements = [tF.makeTransform(geom.AffineTransform(np.random.rand(2))) for _ in range(n_stamps)]
     stamp_list = [stamps.Stamp(stamp_im=stampIm,
                                position=geom.SpherePoint(geom.Angle(ra, geom.degrees),
-                                                         geom.Angle(dec, geom.degrees)))
-                  for stampIm, ra, dec in zip(stampImages, ras, decs)]
+                                                         geom.Angle(dec, geom.degrees)),
+                               archive_element=ae)
+                  for stampIm, ra, dec, ae in zip(stampImages, ras, decs, archive_elements)]
     metadata = PropertyList()
     metadata['RA_DEG'] = ras
     metadata['DEC_DEG'] = decs
 
-    return stamps.Stamps(stamp_list, metadata=metadata)
+    return stamps.Stamps(stamp_list, metadata=metadata, use_archive=True)
```

Comment:
I think we also need a test that uses the default of `use_archive=False`.  I don't think we necessarily need a duplicate test for True and False for all tests, but maybe test with the default in most tests and have a specific test that uses True?